diffs/filebeat.diff

```diff
@@ -0,0 +1,4239 @@
+--- /dev/fd/63
++++ html/en/beats/filebeat/master/configuration-autodiscover-hints.html
+@@ -179,7 +179,7 @@
+            input. You can use hints to modify this behavior. This is the full list of supported hints:
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_logs_enabled_literal">
++           <a id="_co_elastic_logsenabled">
+            </a>
+            <code class="literal">
+             co.elastic.logs/enabled
+@@ -200,7 +200,7 @@
+            .
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_logs_multiline_literal">
++           <a id="_co_elastic_logsmultiline">
+            </a>
+            <code class="literal">
+             co.elastic.logs/multiline.*
+@@ -219,7 +219,7 @@
+            for a full list of all supported options.
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_logs_include_lines_literal">
++           <a id="_co_elastic_logsinclude_lines">
+            </a>
+            <code class="literal">
+             co.elastic.logs/include_lines
+@@ -238,7 +238,7 @@
+            for more info.
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_logs_exclude_lines_literal">
++           <a id="_co_elastic_logsexclude_lines">
+            </a>
+            <code class="literal">
+             co.elastic.logs/exclude_lines
+@@ -257,7 +257,7 @@
+            for more info.
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_logs_module_literal">
++           <a id="_co_elastic_logsmodule">
+            </a>
+            <code class="literal">
+             co.elastic.logs/module
+@@ -278,7 +278,7 @@
+            for the list of supported modules.
+           </p>
+           <h5>
+-           <a id="_literal_co_elastic_logs_fileset_literal">
++           <a id="_co_elastic_logsfileset">
+            </a>
+            <code class="literal">
+             co.elastic.logs/fileset
+@@ -301,7 +301,7 @@
+ co.elastic.logs/fileset.stderr: error</pre>
+           </div>
+           <h5>
+-           <a id="_literal_co_elastic_logs_raw_literal">
++           <a id="_co_elastic_logsraw">
+            </a>
+            <code class="literal">
+             co.elastic.logs/raw
+@@ -322,10 +322,10 @@
+            overrides every other hint and can be used to create both a single or a list of configurations.
+           </p>
+           <div class="pre_wrapper lang-yaml">
+-           <pre class="programlisting prettyprint lang-yaml">co.elastic.logs/raw: "[{"containers\":{"ids\":[\"${data.container.id}"]},\"multiline\":{"negate\":\"true\",\"pattern\":\"^test\"},\"type\":\"docker\"}]"</pre>
+-          </div>
+-          <h5>
+-           <a id="_literal_co_elastic_logs_processors_literal">
++           <pre class="programlisting prettyprint lang-yaml">co.elastic.logs/raw: "[{\"containers\":{\"ids\":[\"${data.container.id}\"]},\"multiline\":{\"negate\":\"true\",\"pattern\":\"^test\"},\"type\":\"docker\"}]"</pre>

```

Comment:
fixed in https://github.com/elastic/beats/commit/bdf48e12a8c520856cb3de51a749460e0c80d6a7